.github/workflows/release_image.yml

```diff
@@ -0,0 +1,89 @@
+name: Release Images
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: 'Infinispan Server version'
+        required: true
+        type: string
+      branch:
+        description: 'The repository branch to checkout'
+        required: true
+        type: string
+      type:
+        description: 'The type of release (latest, prerelease)'
+        required: true
+        type: string
+      push:
+        description: 'Push to remote repositories'
+        default: true
+        type: boolean
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Infinispan Server version'
+        required: true
+        type: string
+      branch:
+        description: 'The repository branch to checkout'
+        default: main
+        type: string
+      type:
+        description: The type of release
+        required: true
+        default: default
```

Comment:
Good job the GH lint failed. We need to re-add "default" as an option as we need this for releases that aren't `:latest` but are also not prereleases. In the case of 15.0.x & 15.2.x we will actually be calling the old infinispan-images workflow (`release.yml` needs updating for this as discussed on the triage call), but eventually we will be in this scenario with the various 16.x.y streams.